### DISCUSSION TITLE
Webhook is not posting “time” field data with request object

### DISCUSSION THREAD

**Discussion by abhattvs** [fingerprintjs/fingerprintjs#764] [General]
During our analysis we found that, Webhook is not posting “time” field data with request object. Kindly advise.

![image](https://user-images.githubusercontent.com/104843767/166468085-556e654d-c922-4d46-a7ec-1b5d18cb2aaa.png)

**Comment by makma** [comment]
Thank you @abhattvs, I notified the team.
